scripts/set_modes.sh

```diff
@@ -18,3 +18,25 @@ sudo chmod 0750 ~/domlogs
 sudo su $USER_NAME -c "echo 'export PATH=\$PATH:/usr/local/pgsql/bin' >> /home/$USER_NAME/.bashrc"
 
 echo "modes setup finished"
+
+# 2. turn off and remove apache
+sudo systemctl stop apache2
+sudo systemctl disable apache2
+
+# 3. install nginx
+sudo apt update
+sudo apt install nginx
+sudo ufw allow "Nginx Full"
+
+#4. turn nginx on
+sudo systemctl start nginx
+sudo systemctl restart nginx
+
+#5. init front-end of make profiler
+profile_make_init_viewer -o=/home/$USER_NAME/public_html 
+
+# create symbolic links for necessary files which should be exposed to web.
+ln -s ~/geocint/report.json  ~/public_html/report.json
+ln -s ~/geocint/make.svg  ~/public_html/make.svg
+ln -s ~/geocint/logs  ~/public_html/logs
```

Comment:
do we need to add index.html -> status report also here?

Reply:
the line above does just that
`profile_make_init_viewer -o=/home/$USER_NAME/public_html`

https://github.com/konturio/geocint-runner/pull/43/files#diff-5a0415f0560aaf464883d19d05e3b6be87e1dfbae9019244786af36cd8c81306R36

it publishes index.html and js/css to the public_html folder.